.agents/contributor-skills/dynamo-frontend-benchmark/scripts/extract_throughput.py

```
@@ -0,0 +1,61 @@
+    sys.exit(f"no profile_export.jsonl under {p}")
+
+def main():
+    path = find_jsonl(sys.argv[1])
+    conc = None
+    if '--conc' in sys.argv:
+        conc = int(sys.argv[sys.argv.index('--conc')+1])
+    S, E, lat, ttft = [], [], [], []
+    for line in open(path):
+        if not line.strip():
+            continue
+        r = json.loads(line); m = r['metadata']
```

Ruff (0.15.15)

[error] Multiple statements on one line (semicolon)

(E702)